CMakeLists.txt

```diff
@@ -24,68 +24,72 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
 PROJECT(vsUTCS)
 
-INSTALL(DIRECTORY .vegastrike/ DESTINATION share/vegastrike/.vegastrike
-        PATTERN "*.am" EXCLUDE)
+SET(CPACK_IGNORE_FILES
+    build/
+    packages/
+    \\\\.git/
+    \\\\.vs/
+    \\\\.vscode/
+    ".*\\\\.pyc"
+)
+SET(CPACK_SOURCE_IGNORE_FILES
+    build/
+    packages/
+    \\\\.git/
+    \\\\.vs/
+    \\\\.vscode/
+    ".*\\\\.pyc"
+)
+
+INSTALL(DIRECTORY .vegastrike/ DESTINATION share/vegastrike/.vegastrike)
 
 INSTALL(DIRECTORY ai/ DESTINATION share/vegastrike/ai
         PATTERN "*.am" EXCLUDE)
```

Comment:
your able to remove the `PATTERN "*.am" EXCLUDE` portion from a few lines, could we extend that to all of the lines with this change?

Reply:
@BenjamenMeyer No, unfortunately. I tried it, and it didn't work without `PATTERN "*.am" EXCLUDE` on at least a few lines. It brought in Makefile.am files, which evidently should be excluded.

Reply:
@stephengtuggy thanks!